src/commonTest/kotlin/baaahs/show/LayoutValidatorSpec.kt

```diff
@@ -0,0 +1,63 @@
+package baaahs.show
+
+import baaahs.describe
+import baaahs.gl.override
+import baaahs.toBeSpecified
+import ch.tutteli.atrium.api.fluent.en_GB.containsExactly
+import ch.tutteli.atrium.api.fluent.en_GB.isEmpty
+import ch.tutteli.atrium.api.verbs.expect
+import org.spekframework.spek2.Spek
+
+object LayoutValidatorSpec : Spek({
+    describe<LayoutValidator> {
+        val layoutValidator by value { LayoutValidator() }
+        val tab by value<Tab> { toBeSpecified() }
+        val invalidRegions by value { layoutValidator.findInvalidRegions(tab) }
+
+        context("when all regions are single cells") {
+            override(tab) { tab(3, 3, "ABC DEF GHI") }
```

Comment:
This makes it behave as though line 14 had this value the whole time, within this `context`.